### ISSUE TITLE
Instalation problem

### ISSUE THREAD

**Issue by nerijunior** [adrenth/redirect#23]
During the install process:
```
SQLSTATE[42000]: Syntax error or access violation: 1067 Invalid default value for 'timestamp' (SQL: create table `adrenth_redirect_clients` (`id` int unsigned not null auto_increment primary key, `redirect_id` int unsigned not null, `timestamp` timestamp default 0 not null, `day` tinyint unsigned not null, `month` tinyint unsigned not null, `year` smallint unsigned not null, `crawler` varchar(255) null) default character set utf8 collate utf8_unicode_ci engine = InnoDB)" on line 662 of ...lluminate/Database/Connection.php"
```

**Comment by adrenth** [comment]
This problem is related to this release note: http://octobercms.com/support/article/rn-4

Which can be simply fixed by adding `'strict' => false` to your `config/database.php` file in the `mysql` section.

But as a plugin author I need to fix this issue for my plugin, which of course I do at the next release of this plugin ;-) Thanks for reporting this issue.

**Comment by adrenth** [comment]
Fixed in 2.0.8.

**Comment by nerijunior** [comment]
Really thanks by the fix! Awesome plugin.

Good job.
